parl/remote/remote_decorator.py

```diff
@@ -179,8 +179,64 @@ def request_cpu_resource(self, global_client, max_memory):
                     cnt -= 1
                 return None
 
+            def __setattr__(self, attr, value):
+                if attr not in cls().__dict__:
```

Comment:
We should not initialize an object here, please put this piece of logic in the remote object.

Reply:
fixed